src/cli/commands/init.js

```diff
@@ -136,6 +138,9 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
         }
       } else {
         answer = (await reporter.question(question)) || def;
+        if (entry.isShorthand && entry.isShorthand(answer)) {
+          answer = 'https://github.com/' + answer;
+        }
```

Comment:
>Please use backticks when concatenating strings :)

We may wanna enable the following ESLint rules then:

- http://eslint.org/docs/rules/no-useless-concat
- http://eslint.org/docs/rules/prefer-template